README.md

```diff
@@ -0,0 +1,2 @@
+# HTML-CSS-project-Trials-1
+Trials 1 Project
```

Comment:
- [ ] 💡 Unfortunately, you have submitted an unprofessional README.m file.

✨ The documentation of a project will help us to manage it, among many things such as making the right decisions, supported by specific data. Some of the benefits that we can find when we properly document a project are:
*Illustrates the project with the steps to follow, as well as all the elements involved in it. 🔍
*It guides and trains collaborators. 👤
*It reduces the risk of ambiguities and confusion that could jeopardize the project. 😕
You can use the following elements to structure your README document:
📖 About the project
💻 Getting star
👤 Authors
🤝 Contributing
🙏 Acknowledgements
❓ FAQ
✍️ License
etc.
Or you can also use the README template from Microverse 👍
https://github.com/microverseinc/readme-template